### ISSUE TITLE
Manipulation of the filter parameters before processing

### ISSUE THREAD

**Issue by Kuhlig** [mevdschee/php-crud-api#937]
Hello 

I can see in the log files that another tool (make.com) is sending the following request 

` /records/exam_sets?filter=uuid%2Ceq%2C54dd6e8c"`

but I would need (as you know)

`/records/exam_sets?filter=uuid,eq,54dd6e8c`

Is there any way to use STR_REPLACE (or something else) to manipulate the records before processing?

Matching handlers? - I would try, but that's not an easy (and fast) way for me to do it
Validation of inputs - I don't think so - it's just a check.
Maybe type sanitation but ...

Thanks for the help 
rob

**Comment by Kuhlig** [comment]
> What is the stray " doing 
I think this si just a copy error from the logfile. i will try again an report back

**Comment by Kuhlig** [comment]
Hallo Maurits

Thanks for your quick reply. I can NOT reproduce it. Although I tested it several times before, I think I ran into a problem with make.com.

If you specify the filter parameter twice for different table columns it sends only the last parameter. Therefore I got all Exam Sets, but since I only waited for 1 Exam Set I considered this as an error. In fact I got only ExamSets of type Sample so the 2nd filter worked.  but the first defined one not. This confused me, sorry that I made you work.

The service offers the following option 

`By default, Make handles multiple values for the same url query string parameter key as arrays (e.g. [www.test.com?foo=bar&foo=bazwill](http://www.test.com/?foo=bar&foo=bazwill) be converted to [www.test.com?foo[0]=bar&foo[1]=baz](http://www.test.com/?foo[0]=bar&foo[1]=baz)). To disable this feature, activate this option.`

As I said when I turned it off only the second parameter comes with. I have it reported there, could be considered a BUG, right?
is there a way to process an array here?

Thanks rob
